skops/card/tests/test_card.py

```diff
@@ -56,4 +44,54 @@ def test_plot_model():
         model_card.save(os.path.join(f"{dir_path}", "README.md"))
         with open(os.path.join(f"{dir_path}", "README.md"), "r") as f:
             model_card = f.read()
-        assert "<style>" in model_card
+            assert "<style>" in model_card
```

Comment:
I know this test has already been there but still: Is this really a sufficient test?

Reply:
What would you do to assert validity of HTML? I wanted to make this lightweight. 
One think I thought of was to have the content of the HTML somewhere else, load and assert equality.

Reply:
A style tag could be in the model card for other reasons, I wonder if there is something more specific we could test.

To test such things **super accurately**, we would probably have to parse the document into a tree or something and then query its contents, but I agree that this is not really lightweight.

Reply:
I rather not parse the whole thing, but we could look for something more specific, like a parameter name or a class name.